source/codegen/metadata/nirfmxinstr/functions.py

```diff
@@ -1314,6 +1314,36 @@
         ],
         'returns': 'int32'
     },
+    'GetAvailablePaths': {
```

Comment:
This has one of the changes from https://github.com/ni/grpc-device-scrapigen/pull/197 but not the others. Are you planning to update this one or make a separate pr?

Reply:
https://github.com/ni/grpc-device/pull/990
https://github.com/ni/grpc-device/pull/981 
rest of the function are already added